style.css

```diff
@@ -0,0 +1,124 @@
+body {
+  font: 16px/1.5 Arial, Helvetica, sans-serif;
+}
+
+* {
+  text-decoration: none;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  min-height: 50px;
+}
+
+#branding {
+  font-size: 15px;
+  color: rgb(75, 75, 216);
+}
+
+.container {
+  margin: auto;
+  padding-top: 15px;
+  width: 80%;
+  height: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav li {
+  display: inline;
+  padding-right: 20px;
+}
+
+a {
+  color: #000;
+}
+
+.showcase {
+  border-bottom-left-radius: 100px;
+  margin-bottom: 10vh;
+}
+
+.hero {
+  width: 60%;
+  margin: auto;
+}
+
+.hero h1 {
+  font-size: 50px;
+  font-weight: bold;
+  line-height: normal;
+}
+
```

Comment:
- [ ] Please, consider that the content of your headline section is aligned with flexbox